Tasks/Podvornaya/Task5/Task5.py

```diff
@@ -0,0 +1,57 @@
+import math
+
+
+def check_str(string):
+    string = string.replace(" ", "")
+    string = [x for x in string]
```

Comment:
list(string) would be much faster, you can read some details here - https://stackoverflow.com/questions/29356846/python-list-vs-list-comprehension-building-speed

comprehensions mostly considered faster than a for loop, not a list conversion, and you could always iterate through a string itself without converting it